static/js/controllers/edit-contact.js

```diff
@@ -86,6 +92,10 @@ var libphonenumber = require('libphonenumber/utils'),
           };
           $scope.category = $scope.contact.type === 'person' ? 'person' : 'place';
           $scope.contactId = null;
+          if ($scope.category === 'place') {
+            // set the Primary Contact field for places to be 'blank'
+            $scope.contact.contact = $scope.parents[0];
```

Comment:
1. When the contact is saved is the contact.contact set to the null contact?
2. Yes - at the same level as the populateParents function?